src/robot_interface/models/exceptions/__init__.py

```diff
@@ -1,7 +0,0 @@
-from .robot_exceptions import (
-    RobotCommunicationException,
-    RobotException,
-    RobotInfeasibleMissionException,
-    RobotInfeasibleStepException,
-    RobotInvalidResponseException,
-)
```

Comment:
Why remove these? Circular dependency exception?

Reply:
Yes. 

Also, the pattern which is adding these into `__init__.py` is not friendly towards refactoring activities. It causes some extra import statements but that is handled automatically through packages. So after a discussion with @tsundvoll we decided to remove this.

Reply:
I agreed that having re-exports in __init__ is not always so refactor-friendly. Really hate to do
`from .../exceptions/exceptions import Exception`, but I will allow it.

Reply:
If you have an IDE that handles it for you you won't hate it as much as you don't look at it :D